pkgs/os-specific/darwin/apple-source-releases/adv_cmds/xcode.nix

```diff
@@ -0,0 +1,21 @@
+{stdenv, fetchurl, xcbuild}:
+
+stdenv.mkDerivation {
```

Comment:
Does this expression actually build? Amazeballs if so, and it'll make our lives so much easier.